src/scripts/main.js

```diff
@@ -1,5 +1,34 @@
 'use strict';
 
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
 document.addEventListener('click', (e) => {
-  // write code here
+  const wallCoords = wall.getBoundingClientRect();
+  const spiderCoords = spider.getBoundingClientRect();
+  const border = parseInt(
+    window.getComputedStyle(wall).borderWidth.split('px').join(''),
+  );
```

Comment:
The calculation of the border width assumes that the border is specified in pixels. If the border is not specified in pixels, this could lead to incorrect calculations. Ensure that the border width is always in pixels or handle other units appropriately.